Scripts/Python/hopsan.py

```diff
@@ -0,0 +1,52 @@
+class hopsan:
+    def __init__(self):
+        import ctypes
+        import os
+        self.hdll = ctypes.cdll.LoadLibrary(os.getenv('HOPSANHOME')+"/bin/libhopsanc.so")
+        libpath= os.getenv('HOPSANHOME')+"componentLibraries/defaultLibrary/libdefaultcomponentlibrary.so"
```

Comment:
This assumes Linux, maybe support windows as well

Reply:
I added Windows support. Not sure if we need to load the default library on Windows, but I added it just in case. The worst thing that can happen is that it complains that the components are already loaded.

Reply:
True, I keep forgetting that the default library is built in on Windows.
But in this case the .dll should not even be present.
What will happen then

Reply:
I think the best solution would be for the code to check if the defaultlibrary.dll can be found and then load it, else not try to load it. Then it would support both cases. The official release does not include that defaultlibrary dll

Reply:
I added a check for this. It also required an additional function for printing waiting messages, so that the Hopsan version information is still being printed when loading the library.